### ISSUE TITLE
NTR: Small intestine biopsy: thinning of mucosa and crypts (Lab: endoscopy and exploratory surgery)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8664]
New term request Small intestine biopsy: thinning of mucosa and crypts (nadea3_200428190429):

**Comment by pnrobinson** [comment]
@MickeySegal
can you provide the disease context or a PMID?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3099351/ "Analysis of small bowel biopsies in elderly patients with bacterial overgrowth revealed blunting of the intestinal villi, thinning of the mucosa and crypts, and increased intraepithelial lymphocytes, all of which reversed with antibiotic treatment"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000337
- label: Thinned small intestinal mucosa
- parents: [Abnormal small intestinal mucosa morphology (HP:0025129)](https://hpo.jax.org/app/browse/term/HP:0025129)
- pmid: 
- def: Abrnormal reduction in the height of the mucosa of the small intestine. The mucosa is arranged in regular circular folds (the circular valves of Kerckrin), the mucosal surface is covered with small projections called villi, which are slender, about 1.5-2 mm long projections of the mucosa into the lumen. In between the villi are the crypts. This term refers to a pathological thinning of these structures.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8664
- synonyms:
